# HG changeset patch
# User Rik <rik@octave.org>
# Date 1462294757 25200
#      Tue May 03 09:59:17 2016 -0700
# Node ID c579ab9ee4f376470397eee91365f23fff39454a
# Parent  927d7bc4bdfa53ce78be55ff49eaa21078863e83
# Parent  32a6d948fea7d60565c8f5043d5441e830444a91
maint: Periodic merge of stable to default.

diff --git a/libinterp/corefcn/syscalls.cc b/libinterp/corefcn/syscalls.cc
--- a/libinterp/corefcn/syscalls.cc
+++ b/libinterp/corefcn/syscalls.cc
@@ -135,17 +135,17 @@ DEFUNX ("exec", Fexec, args, ,
 @deftypefn {} {[@var{err}, @var{msg}] =} exec (@var{file}, @var{args})\n\
 Replace current process with a new process.\n\
 \n\
 Calling @code{exec} without first calling @code{fork} will terminate your\n\
 current Octave process and replace it with the program named by @var{file}.\n\
 For example,\n\
 \n\
 @example\n\
-exec (\"ls\" \"-l\")\n\
+exec (\"ls\", \"-l\")\n\
 @end example\n\
 \n\
 @noindent\n\
 will run @code{ls} and return you to your shell prompt.\n\
 \n\
 If successful, @code{exec} does not return.  If @code{exec} does return,\n\
 @var{err} will be nonzero, and @var{msg} will contain a system-dependent\n\
 error message.\n\
